### PR TITLE
Update CREDITS

### PR DESCRIPTION
**Pull request by erlend-sh**

### CODE REVIEW

**Comment by erlend-sh** [comment]
Jacob wanted to include his artist title, Emerald Jak.

@SirGiraffey wanted to include his full name: Michał Grzegórzek - Kiaszewicz

Not entirely sure how to deal with the dash, since we’re already using that for separation.

**Comment by SirGiraffey** [comment]
If the dash is a problem you can remove it, no problem

File: CREDITS.md [hunk 1]
```diff
@@ -18,7 +18,7 @@
 - Carlo Supina, [@cdsupina](https://github.com/cdsupina) – Dev Advocacy
 - Micah, [@tigleym](https://github.com/tigleym) – Dev Advocacy
 - Drake, [@Drakeyn](https://github.com/Drakeyn)– Media Content
-- Zyrafa, [@SirGiraffey](https://github.com/SirGiraffey) – Gameplay Design
+- Michał Grzegórzek - Kiaszewicz, [@SirGiraffey](https://github.com/SirGiraffey) – Gameplay Design
```
Comment (tigleym) [inline CREDITS.md:21]: It looks like the 'ł' character is also missing from the current font set we're using. #316 introduces a new font for the credits this should be fine once it's merged.

**Review by tigleym — APPROVED** [review]
LGTM! Quickly testing this and it looks like we'll have to adjust the x-offset a bit to accomodate smaller window sizes (the text end up getting clipped off screen for longer labels). But I think that can be addressed in another PR.

**Pull request merged**